### DISCUSSION TITLE
Maiana and MacArthur hat versions compatibility

### DISCUSSION THREAD

**Discussion by kagouraki** [peterantypas/maiana#145] [Q&A] [answered]
Hello, 

I have a maiana version 10.5.0 board and a macArthur hat version 1.2.
Can someone tell me if i can connect directly without the breakout board ?
I will make the Ethernet cable myself so if i need to change the standard Ethernet pin-out i can do it.
I am not interested to program mmsi, update firmware and tx off through software. 
Thnks !

**Comment by sailoog** [comment]
Yes, you can: https://macarthur-hat-documentation.readthedocs.io/en/latest/maiana.html

**Comment by kagouraki** [comment]
With the normal ethernet cable pinout (pc to router style) ? 
And just for reference, does it support all the features ? 
Program mmsi, update firmware and software tx off switch

**Comment by sailoog** [comment, marked as answer]
Same pinout as the breakout board, actually the circuits are identical. All the features too.
